Sources/HaCWebsiteLib/ConstitutionManager.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import Dispatch
+import DotEnv
+
+public struct ConstitutionManager {
+  /// The list of workshops as derived from the workshops directory last time it was updated
+  public static var mdConstitution: String = "For some reason the constitution cannot be displayed, please contact the site administrators."
+
+  /// Used to avoid multiple git pulls being run simultaneously
+  private static let serialQueue = DispatchQueue(label: "com.hac.website.const-pull-queue")
```

Comment:
Maybe since we have multiple 'Manager's maintaining their own serial queues, we should abstract this into the GitUtils class? This could be as simple as saying only one git pull can be running at a time or we could say only one git pull into the same directory can be running at a time.